clients/privacy-center/cypress/e2e/consent-banner.cy.ts

```diff
@@ -1438,27 +1409,15 @@ describe("Consent banner", () => {
           analytics: true,
         });
         cy.get("@FidesInitialized")
-          .should("have.been.calledOnce")
-          .its("firstCall.args.0.detail.consent")
-          .should("deep.equal", {
-            data_sales: false,
-            tracking: false,
-            analytics: true,
-          });
-        cy.get("@FidesUpdated")
+          // TODO PROD-1280: Suppress duplicate event if the cookie state is unchanged from an early initialization?
+          // .should("have.been.calledOnce")
```

Comment:
Note that we shouldn't check this in; need to decide what to do and then do it 👍